esinstall/src/index.ts

```diff
@@ -351,6 +352,9 @@ ${colors.dim(
   let isFatalWarningFound = false;
   const inputOptions: InputOptions = {
     input: installEntrypoints,
+    ...(userDefinedRollup.context && {
```

Comment:
instead can we just do `context: userDefinedRollup.context`? Or, does Rollup treat an explicit `undefined` value as literally "set context to `undefined`"?

Reply:
Sure, no problem. The default is `undefined` (implicitly), but making it explicit shouldn't hurt in this case. I just like to be specific as javascript's null/undefined and literally "undefined" as in "property doesn't exist" are (as you know of course) somewhat different depending on how the underlying code checks for these cases. It doesn't seem to cause any issues here tho.